src/fiber/tests/graph.rs

```diff
@@ -364,23 +371,52 @@ fn test_graph_find_path_fee() {
     let mut network = MockNetworkGraph::new(5);
 
     network.add_edge(1, 2, Some(1000), Some(10000));
+    // means node 2 will charge fee_rate 10000 when forwarding tlc
```

Comment:
please add some comments for `add_edge`, specific for the `fee_rate`, which maybe confused in future.